vic/drivers/shared_image/src/vic_init.c

```diff
@@ -792,6 +796,120 @@ vic_init(void)
 
         soil_moisture_from_water_table(&(soil_con[i]), options.Nlayer);
 
+        // Soil thermal node thicknesses and positions
```

Comment:
Just out of curiosity. What happens if we write an state file with one set of thermal node properties/dimensions and try to use that initial state to startup a new simulation with a different set of thermal node properties/dimensions?  Do we raise an error if the run settings do not match the statefile?